python-sdk/tutorials/train-on-low-priority-aml-compute/README.md

```diff
@@ -0,0 +1,13 @@
+---
+page_type: sample
+languages:
+- python
+- azurecli
+products:
+- azure-machine-learning
+description: Learn how to train models using low-priority AML compute.
+---
+
+# Using Low-Priority AML Compute
+
+Learn how to train models using low-priority AML compute.
```

Comment:
You could add some description of what this example does.
For example,
"The example uses Pytorch to train a model to recognize handwritten digits."

Reply:
Thank you! I used this verbatim!